mongoose-bot/example.json

```diff
@@ -0,0 +1,5 @@
+{
+    "token": "",
+    "clientId": "",
+    "guildId": ""
```

Comment:
Need to explain where to get guild id, especially since I wouldn't never guessed that guild is the same thing as server. See https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-#:~:text=Obtaining%20Server%20IDs%20%2D%20Mobile%20App,name%20and%20select%20Copy%20ID.,